src/main/kotlin/no/nav/familie/ks/sak/integrasjon/oppgave/OppgaveService.kt

```diff
@@ -120,6 +121,48 @@ class OppgaveService(
         }
     }
 
+    fun forlengFristÅpneOppgaverPåBehandling(behandlingId: Long, forlengelse: Period) {
+        val dbOppgaver = oppgaveRepository.findByBehandlingIdAndIkkeFerdigstilt(behandlingId)
+
+        dbOppgaver.forEach { dbOppgave ->
+            val gammelOppgave = hentOppgave(dbOppgave.gsakId.toLong())
+            val oppgaveErAvsluttet = gammelOppgave.ferdigstiltTidspunkt != null
+
+            when {
+                gammelOppgave.id == null ->
+                    logger.warn("Finner ikke oppgave ${dbOppgave.gsakId} ved oppdatering av frist")
+
+                gammelOppgave.fristFerdigstillelse == null ->
+                    logger.warn("Oppgave ${dbOppgave.gsakId} har ingen oppgavefrist ved oppdatering av frist")
+
+                oppgaveErAvsluttet -> {}
+                else -> {
+                    val nyFrist = LocalDate.parse(gammelOppgave.fristFerdigstillelse!!).plus(forlengelse)
```

Comment:
```suggestion
                    val nyFrist = LocalDate.parse(gammelOppgave.fristFerdigstillelse).plus(forlengelse)
```
Trenger ikke nullchecke her da det er utført øverst på 135.